example/themes/default/templates/list-page.html

```diff
@@ -4,7 +4,7 @@
         <title>{{.Meta.Title}}</title>
         <meta name="author" content="{{.Meta.Author}}" />
         <meta name="description" content="{{.Meta.Description}}" />
-        <link rel="stylesheet" type="text/css" href="/css/style.css" />
+        <link rel="stylesheet" type="text/css" href="/assets/style.css" />
```

Comment:
I think it would be better to still use a subfolder `assets/css`.
So here it would be `/assets/css/style.css`.